test/agenda.js

```diff
@@ -26,7 +26,7 @@ const jobTimeout = process.env.TRAVIS ? 2500 : 500;
 const jobType = 'do work';
 const jobProcessor = () => {};
 
-describe('Agenda', () => {
+describe('Agenda', function() { // eslint-disable-line prefer-arrow-callback
```

Comment:
Is there a reason not to use arrow function here?

Reply:
To make the bottom this.timeout() go work